examples/moses-pln-synergy/scm/README.md

```diff
@@ -164,6 +179,10 @@ $5 = (SetLink
    )
 )
 
+### (5) Build the tautology that if X takes treatment-1, then
+    treatment-1 contains compound-A. Semi-formally
+    (\x take(x, treatment-1)) -> (\x contain(treatment-1, compound-A))
```

Comment:
I understand what you mean. You are right, saying that a lambda exposes the bound variables to the user is incorrect, what it exposes is an abstraction, and that abstraction contains a variable (which name is irrelevant from the user's view) and a body. I guess from the outside (the user) using an abstraction or combinators would make no difference, so yes I agree.